src/wallet/rpcwallet.cpp

```diff
@@ -2650,22 +2653,48 @@ Value zc_raw_pour(const json_spirit::Array& params, bool fHelp)
         throw runtime_error("unsupported pour input/output counts");
     }
 
-    // TODO: #808
-    uint256 pubKeyHash;
+    uint256 joinSplitPubKey;
```

Comment:
+1 (Since libsodium's constants won't change, I wish they put the value in the name like `crypto_sign_PUBLICKEYBYTES_32`, it would have simplified stuff like this)